packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -42,19 +42,29 @@ def CallDhclient(
     logger.warning('Could not enable interfaces %s.', interfaces)
 
 
-def CallDhclientIpv6(interfaces, logger, dhclient_script=None):
+def CallDhclientIpv6(interfaces, logger, dhclient_script=None, release_lease=False):
   """Configure the network interfaces for IPv6 using dhclient.
 
   Args:
     interface: string, the output device names for enabling IPv6.
     logger: logger object, used to write to SysLog and serial port.
     dhclient_script: string, the path to a dhclient script used by dhclient.
+    release_lease: Release the IPv6 lease.
   """
-  logger.info('Enabling IPv6 on the Ethernet interfaces %s.', interfaces)
+  logger.info('Calling Dhclient for IPv6 configuration '
+              'on the Ethernet interfaces %s.', interfaces)
 
   timeout_command = ['timeout', '5']
   dhclient_command = ['dhclient']
 
+  if release_lease:
+    try:
+      subprocess.check_call(timeout_command + dhclient_command + ['-1', '-6', '-r', '-v']
```

Comment:
Please line wrap after the first open paren.

Reply:
done